pkgs/development/compilers/rust/rust-lib-src.nix

```diff
@@ -1,11 +1,10 @@
-{ stdenv, rustc }:
+{ runCommand, rustc }:
 
-stdenv.mkDerivation {
-  name = "rust-lib-src";
+runCommand "rust-lib-src" {
   src = rustc.src;
-  phases = [ "unpackPhase" "installPhase" ];
+} ''
+  source $stdenv/setup
+  unpackPhase
```

Comment:
Don't source stdenv.

```suggestion
```

Reply:
thought it would be needed for finding `unpackPhase`

Reply:
It is, don't use `unpackPhase` either.

Reply:
:+1: